Gemfile

```diff
@@ -2,6 +2,9 @@ source "https://rubygems.org"
 
 ruby "3.2.0"
 
+# Intertionalization
+gem 'rails-i18n', '~> 7.0.0' 
```

Comment:
No rails 7 a i18 é built-in, não se faz necessário a gem

Reply:
Ok! Anotado e já consertado!